src/app/profiles/App/pathToContributor.ts

```diff
@@ -1,11 +1,12 @@
 import slugify from 'slugify';
 import i18n from 'i18next';
 import { Contributor } from 'libs/domain/contributor';
+import { getPathPrefix } from '../../../libs/facets/getPathPrefix';
 
 const pathToContributor = (contributor: Contributor) => {
-  return `${i18n.t('path.profiles.contributors')}/${contributor.id}/${slugify(
-    contributor.name
-  )}`;
+  return `${getPathPrefix(i18n)}${i18n.t('path.profiles.contributors')}/${
```

Comment:
For complex path joining it would be way cooler to use smthing like node's `path.join`.
Here we don't which part come with a `/` or not...
Maybe smthing could be crafted with the [URL API](https://developer.mozilla.org/fr/docs/Web/API/URL) instead?

Reply:
We can't use `path` since we’re browser side, and `URL` constructor doesn’t seem to have a builin parts joining feature … 

I’m lookinq at https://www.npmjs.com/package/join-path …

Reply:
Nope, it also needs `path` … I’m looking at https://www.npmjs.com/package/join-url

Reply:
Love it!